posix.mak

```diff
@@ -337,7 +337,7 @@ SPEC_ROOT=$(addprefix spec/, \
 SPEC_DD=$(addsuffix .dd,$(SPEC_ROOT))
 
 CHANGELOG_FILES:=$(basename $(subst _pre.dd,.dd,$(wildcard changelog/*.dd)))
-ifndef RELEASE
+ifneq (1,$(RELEASE))
```

Comment:
This allows a user to do sth. like `RELEASE=$(MY_FANCY_RELEAS_VAR)`.
The other macro ifs also use this style.